splipy/volume.py

```diff
@@ -62,7 +62,12 @@ def faces(self):
         :return: Boundary faces
         :rtype: (Surface)
         """
-        return tuple(self.section(*args) for args in sections(3, 2))
+        boundary_faces = [self.section(*args) for args in sections(3, 2)]
+        for i,b in enumerate(self.bases):
+            if b.periodic > -1:
+                boundary_faces[2*i  ] = None
+                boundary_faces[2*i+1] = None
+        return tuple(boundary_faces)
```

Comment:
Can we do `tuple(f for f in boundary_faces if f is not None)`, and then ditch the check for `None` in *stl.py*? Or is there some other code that relies on there being 6 faces or something?

Reply:
I would expect there to be some end-user code with logic built on this returning 6 faces. It is also very helful to know the ordering (i.e. face[2] would *always* correspond to vmin).

The way I see this is that `SplineModel` can ditch the extra faces (both in a multi- and singlepatch setting), but `Volume` objects should return a predictable amount of entries.

Reply:
Yeah, that's fine.